### DISCUSSION TITLE
Using calendar plugin

### DISCUSSION THREAD

**Discussion by vasireddy** [microsoft/semantic-kernel#7129] [Q&A] [answered]
I am learning semantic kernel and was trying to consume calendar plugin.  I am wondering if i can get some sample by community here or point me to resources on how to read outlook calendar events using dotnet/src/Plugins/Plugins.MsGraph/CalendarPlugin.cs

Thank you so much

**Comment by sophialagerkranspandey** [comment, marked as answer]
We'll be cleaning up the plugins and will be removing ones like this. We recommend using MS graph for authentication.
